innovation.css

```diff
@@ -1441,116 +1464,131 @@
     box-shadow: 5px 5px grey;
 }
 
-/* Locations where the square and hexagon icons go on each card */
+/* Locations where the square and hexagon icons go on each card 
+*
+*  Square icons should be ~4% of the card width away from the left/bottom/right edges. It currently is 7px
+*  Square icons should be ~6% of the card width away from the top edge. it is currently 7px
+* 
+*  For the middle square icon it should be centered in the card. 180(card size)/2 - 36(icon size)/2 = 72
+*
+*  The Hex icons do not fill their entire box so we have to fudg some of the offsets
+*/
 
 .M.top_left_icon.square_card_icon {
-    top: 8px;
-    left: 10px;
+    top: 7px;
+    left: 7px;
 }
 
 .M.top_left_icon.hexagon_card_icon {
-    top: 7px;
-    left: 7px;
+    top: 2px;
+    left: 4px;
 }
 
 .M.bottom_left_icon.square_card_icon {
-    top: 87px;
-    left: 10px;
+    bottom: 7px;
+    left: 7px;
 }
 
 .M.bottom_left_icon.hexagon_card_icon {
-    top: 85px;
-    left: 7px;
+    bottom: 1px;
+    left: 4px;
 }
 
 .M.bottom_center_icon.square_card_icon {
-    top: 87px;
-    left: 76px;
+    bottom: 7px;
+    left: 72px;
 }
 
 .M.bottom_center_icon.hexagon_card_icon {
-    top: 85px;
-    left: 73px;
+    bottom: 1px;
+    left: 69px;
 }
 
 .M.bottom_right_icon.square_card_icon {
-    top: 87px;
-    left: 140px;
-
+    bottom: 7px;
+    right: 7px;
 }
 
 .M.bottom_right_icon.hexagon_card_icon {
-    top: 85px;
-    left: 137px;
-
+    bottom: 1px;
+    right: 2px;
 }
 
-/* Square card icons (most cards have 3 large resource icons on them) */
+/* Square card icons (most cards have 3 large resource icons on them) 
+*
+*  These properties of the square icon are directly related. Right now they are made to be 20% of the card width.
+*  { Background-size, width, height, background-position-x, background-position-y }
+*/
 
 .square_card_icon {
     background-image: url('img/resource_icons.jpg');
-    background-size: 180px auto;
+    background-size: 230px auto;
     position: absolute;
 }
 
 .M.square_card_icon {
-    width: 28px;
-    height: 28px;
+    width: 36px;
+    height: 36px;
 }
 
 .M.icon_1.square_card_icon  {
     background-position-x: -1px;
 }
 
 .M.icon_2.square_card_icon  {
-    background-position-x: -31px;
+    background-position-x: -39px;
 }
 
 .M.icon_3.square_card_icon  {
-    background-position-x: -61px;
+    background-position-x: -78px;
 }
 
 .M.icon_4.square_card_icon  {
-    background-position-x: -91px;
+    background-position-x: -116px;
 }
 
 .M.icon_5.square_card_icon  {
-    background-position-x: -121px;
+    background-position-x: -155px;
 }
 
 .M.icon_6.square_card_icon  {
-    background-position-x: -151px;
+    background-position-x: -192.5px;
 }
 
 .M.color_0.square_card_icon  {
     background-position-y: -1px;
 }
 
 .M.color_1.square_card_icon  {
-    background-position-y: -31px;
+    background-position-y: -39px;
 }
 
 .M.color_2.square_card_icon  {
-    background-position-y: -61px;
+    background-position-y: -77px;
```

Comment:
I think this should be -78px. See the "Clothing" card.

Reply:
agreed